test/grove-mainnet-fork/LayerZero.t.sol

```diff
@@ -0,0 +1,588 @@
+
+import {LZBridgeTesting} from "xchain-helpers/testing/bridges/LZBridgeTesting.sol";
+import {LZForwarder} from "xchain-helpers/forwarders/LZForwarder.sol";
+
+import {ForeignControllerDeploy} from "../../deploy/ControllerDeploy.sol";
+import {ControllerInstance} from "../../deploy/ControllerInstance.sol";
+
+import {ForeignControllerInit} from "../../deploy/ForeignControllerInit.sol";
+
+import {ALMProxy} from "../../src/ALMProxy.sol";
+import {ForeignController} from "../../src/ForeignController.sol";
+import {RateLimits} from "../../src/RateLimits.sol";
+import {RateLimitHelpers} from "../../src/RateLimitHelpers.sol";
+
+import "./ForkTestBase.t.sol";
+
+contract PlasmaChainUSDTToLayerZeroTestBase is ForkTestBase {
+    using DomainHelpers for *;
+    using LZBridgeTesting for Bridge;
+
+    /**********************************************************************************************/
+    /*** Constants/state variables                                                              ***/
+    /**********************************************************************************************/
+
+    address pocket = makeAddr("pocket");
+
+
+    /**********************************************************************************************/
+    /*** Plasma addresses                                                                         ***/
+    /**********************************************************************************************/
+
+    // Plasma OUpgradeable USDT OFT
+    address constant USDT0_OFT_PLASMA_ADDRESS = 0x02ca37966753bDdDf11216B73B16C1dE756A7CF9;
+    // Plasma USDT0
+    address constant USDT0_PLASMA_ADDRESS = 0xB8CE59FC3717ada4C02eaDF9682A9e934F625ebb;
+
+    /**********************************************************************************************/
+    /*** ALM system deployments                                                                 ***/
+    /**********************************************************************************************/
+
+    ALMProxy foreignAlmProxy;
+    RateLimits foreignRateLimits;
+    ForeignController foreignController;
+
+    /**********************************************************************************************/
+    /*** Casted addresses for testing                                                           ***/
+    /**********************************************************************************************/
+
+    // Mainnet OFTs
+    IERC20 usdtOft;
+
+    // Plasma Tokens
+    IERC20 usdsPlasma;
+    IERC20 susdsPlasma;
+    IERC20 usdt0Plasma;
+    IERC20 usdt0OftPlasma;
+
+    IPSM3 psmPlasma;
+
+    uint256 USDT0_PLASMA_SUPPLY; // Total supply of USDT0 on Plasma
+    uint256 USDT0_MAINNET_BALANCE_BEFORE; // How much USDT is in the mainnet OFT contract before the test
+
+    uint32 constant sourceEndpointId = 30101; // Ethereum EID
+    uint32 constant destinationEndpointId = 30383; // Plasma EID
```

Comment:
not necessary for this one, but for the refactor and future reference, maker convention is to line up the `=` signs for groups of variables.